src/ams.py

```diff
@@ -230,6 +247,7 @@ def _get_config(self) -> dict:
 
     def _set_config_item(self, name, value):
         subprocess.run(["/snap/bin/amc", "config", "set", name, value], check=True)
+        logger.info("Set ams configuration item: %s=%s", name, value)
```

Comment:
This may leak the credential info in the logging system.

Reply:
yeah, good spot. I wanted it to be debug level anyway but should I just get rid of it or make an exception for a few config items `image.auth` etc.?

Reply:
I would leave this out or make it only print the key